src/ape/contracts/base.py

```diff
@@ -468,6 +468,46 @@ def __len__(self):
         logs = self.provider.get_contract_logs(self.log_filter)
         return sum(1 for _ in logs)
 
+    def __call__(self, *args: Any, **kwargs: Any) -> MockContractLog:
+        # Create a dictionary from the positional arguments
+        event_args = dict(zip((input.name for input in self.abi.inputs), args))
+
+        overlapping_keys = set(k for k in event_args.keys() if k is not None) & set(
+            k for k in kwargs.keys() if k is not None
+        )
+
+        if overlapping_keys:
+            raise ValueError(
+                f"Overlapping keys found in arguments: '{', '.join(overlapping_keys)}'."
+            )
+
+        # Update event_args with keyword arguments
+        event_args.update(kwargs)  # type: ignore[arg-type]
+
+        # Check that event_args.keys() is a subset of the expected input names
+        if unknown_input_names := set(event_args.keys()) - {
+            input.name for input in self.abi.inputs
+        }:
+            raise ValueError(
+                f"Invalid argument keys found, expected a subset of {unknown_input_names}"
+            )
+
+        # Convert the arguments using the conversion manager
+        converted_args = {}
+        for key, value in event_args.items():
+            if value is None:
+                continue
+            input_abi = next(input for input in self.abi.inputs if input.name == key)
+            ecosystem = self.provider.network.ecosystem
+            type = ecosystem.get_python_types(input_abi)
```

Comment:
it is not necessary to overwrite python builtin keywords here

```suggestion
            py_type = ecosystem.get_python_types(input_abi)
```